From 40c97b4ef4d0cb5ff8a83b8b88ff38afd8daee30 Mon Sep 17 00:00:00 2001
From: Brendan King <Brendan.King@imgtec.com>
Date: Tue, 14 Jan 2020 11:14:01 +0000
Subject: [PATCH 10/19] meson: create xfree86 log directory

Create the xfree86 log directory, just as autotools does. The X Server
will not start if the directory doesn't exist.
---
 hw/xfree86/meson.build  | 2 ++
 hw/xfree86/mklog_dir.sh | 3 +++
 2 files changed, 5 insertions(+)
 create mode 100755 hw/xfree86/mklog_dir.sh

diff --git a/hw/xfree86/meson.build b/hw/xfree86/meson.build
index c4db85c..8d09356 100644
--- a/hw/xfree86/meson.build
+++ b/hw/xfree86/meson.build
@@ -232,3 +232,5 @@ install_man(configure_file(
 ))
 
 subdir('doc')
+
+meson.add_install_script('mklog_dir.sh', log_dir)
diff --git a/hw/xfree86/mklog_dir.sh b/hw/xfree86/mklog_dir.sh
new file mode 100755
index 0000000..a2beae5
--- /dev/null
+++ b/hw/xfree86/mklog_dir.sh
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+[ -n "$1" ] && mkdir -p "${DESTDIR}/$1"
-- 
2.25.1

